app/src/main/java/io/appium/uiautomator2/handler/FindElements.java

```diff
@@ -121,6 +122,16 @@ public AppiumResponse safeHandle(IHttpRequest request) {
 
     private List<Object> findElements(By by) throws ElementNotFoundException, ParserConfigurationException, ClassNotFoundException, InvalidSelectorException, UiAutomator2Exception {
         if (by instanceof By.ById) {
+            String locator = by.getElementLocator();
```

Comment:
This whole section is repeated from `FindElement`. Is there no way to use the version from there without copying it?